app/Http/Controllers/Service/Admin/SponsorsController.php

```diff
@@ -245,7 +245,7 @@ public static function socialPrescribingOverrides()
                 "name" => "DeductFromCarer",
                 "value" => -7,
                 "purpose" => "credits",
-                "entity" => "App\Child",
+                "entity" => "App\Family",
```

Comment:
aha! good spot

Reply:
this entire array feels like it should be a data item in `config/arc`
similarly the rules in the evaluatorfactory

we should make a card to extract them